docker/run.sh

```diff
@@ -5,7 +5,7 @@
 docker run \
     -ti \
     -p 3000:3000 \
-    -e 'SMTP_PASSWORD=smtp_password_goes_here' \
+    -e 'SMTP_PASSWORD=smtp_password_goes_here' -e 'WEB_HOST=http://localhost:3000' \
```

Comment:
Is this always the case? Well, I think that's only used to test locally right?

Reply:
Yes, exactly, you'll still need to change it for the server. This is just for running the docker image locally